STYLEGUIDE.adoc

```diff
@@ -0,0 +1,56 @@
+= Jenkins Documentation Style Guide
+
+CAUTION: This document is still a work in progress.
+
+== Terminology
+
+* Build
+* Job
+* Agent
+* Master
+* Node
+* Pipeline
+
+Mixing of these terms is incorrect
+
+== Assorted comments
+
+* Prefer "for example" over "e.g." which can be more clear to non-native english
+  readers
+* Don't use unordered lists (bullets) in place of section headers. Section
+  headers offer a nesting/association of content in a way lists cannot
+* If you write a sentence such as "there are three ways to do this:" and then
+  intend to follow that statement with a list, use a *numbered* list
+** If you are providing a list of "three ways", each description of a "way"
+should follow the same structure, for example:
+
+[quote]
+----
+
+----
```

Comment:
Whoops! This is what happens with stream of conscious notes and you context switch :)